### PR TITLE
Initial readme with exercises we wanted

### PR DESCRIPTION
**Pull request by linsun**

### CODE REVIEW

**Comment by linsun** [comment]
@duglin @jkomg can you help review this?

**Comment by duglin** [comment]
@linsun can you add placeholder files so the href checker tool isn't unhappy?

File: workshop/README.md [hunk 1]
```diff
@@ -1 +1,18 @@
...
+# Istio 101 workshop
+
+## Workshop Setup
+- [Exercise 1 - Accessing a Kubernetes cluster with IBM Cloud Container Service](exercise-1/README.md)
+
+If you are not familar with Kubernetes, we recommend you to follow [Kubernetes 101](https://github.com/IBM/kube101) learn Kubernetes.  If you are already familar with Kubernetes, feel free to proceed to exercises below to learn Istio.
+
+## Creating a Service Mesh with Istio
+
+- [Exercise 2 - Installing Istio](exercise-2/README.md)
```
Comment (duglin) [inline workshop/README.md]: these should be under the "workshop" folder, e.g. ` workshop/exercise-2/README.md`

**Comment by linsun** [comment]
@duglin sure, I didn't realize we had a href check as I'm creating these PRs from the website.  I'll fix it.

File: workshop/README.md (hunk 1 above)
Comment (linsun) [reply workshop/README.md]: yes, it is planned to be there

**Comment by linsun** [comment]
any idea why i can't view the branch linsun:patch-1?  @duglin @jkomg I also need permission to merge for this repo... many thanks.

**Pull request merged**